Anlab.Mvc/Services/OrderMessageService.cs

```diff
@@ -75,23 +74,32 @@ public OrderMessageService(ViewRenderService viewRenderService, IMailService mai
             _mailService.EnqueueMessage(message);
         }
 
-        public async Task EnqueueCompletedMessage(Order order)
+        public async Task EnqueueAcceptedMessage(Order order)
```

Comment:
@laholstege i'm not sure if we are going to send an email for this case, but for now we can leave it in